tool_collections/vcflib/vcfaddinfo/vcfaddinfo.xml

```diff
@@ -0,0 +1,31 @@
+<tool id="vcfaddinfo" name="VCFaddinfo:" version="0.0.2">
+<requirements>
+    <requirement type="package" version="8a5602bf07">vcflib</requirement>
+</requirements>
+  <description>Adds info fields from the second dataset which are not present in the first dataset</description>

```

Comment:
I think a macro zourld be a good idea here, but we can also work on this after merging.